src/UI/Component/Image/Image.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Image;
+
+/**
+ * This describes how a glyph could be modified during construction of UI.
+ *
+ * Interface Image
+ * @package ILIAS\UI\Component\Image
+ */
+interface Image extends \ILIAS\UI\Component\Component {
+	/**
+	 * Types of images
+	 */
+	const STANDARD = "standard";
+	const RESPONSIVE = "responsive";
```

Comment:
I'm not sure if that is the right direction here. I could imagine easily that a standard image will get other properties then the responsive image over time, so maybe split those in two distinct classes already?